### PR TITLE
[TT-4696] Getting Started docs for tyk-operator (Access API)

### PR DESCRIPTION
**Pull request by buraksekili**
## Description

This PR adds a new section to the [Getting Started](https://github.com/TykTechnologies/tyk-operator/blob/master/docs/getting-started.md) doc to provide details of how users can access their APIs through Tyk Operator.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [ ] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `gofmt -s -w .`
  - [ ] `go vet ./...`
  - [ ] `golangci-lint run`

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -172,3 +173,215 @@ $ curl -i localhost:8080/httpbin/get
...
+        strip_listen_path: true
+EOF
+
+apidefinition.tyk.tyk.io/httpbin configured
+```
+
+In order to access `httpbin` API, you need to include a key to the header.
+
+> **Note**: All supported authentication types by Tyk Operator are listed [here](https://github.com/TykTechnologies/tyk-operator/blob/master/docs/api_definitions.md#client-to-gateway-authentication).
+Authentication token is the default one if you are using keyless API.
```
Comment (komalsukhani) [inline docs/getting-started.md:235]: ``
```suggestion
Authentication token is the default one if you have set `use_keyless` to false and haven't specified any other Authentication mode.
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -172,3 +173,215 @@ $ curl -i localhost:8080/httpbin/get
...
+    "access_rights": {
+      "{API-ID}": {
+        "api_id": "{API-ID}",
+        "api_name": "{API-NAME}",
+        "versions": ["Default"]
+      }
+    },
+    "meta_data": {}
+  }' http://localhost:8080/tyk/keys/create | python -mjson.tool
+{
```
Comment (komalsukhani) [inline docs/getting-started.md:312]: Move response in separate section so user is able to copy just the curl request.

File: docs/getting-started.md [hunk 3]
```diff
@@ -172,3 +173,215 @@ $ curl -i localhost:8080/httpbin/get
...
+```
+
+### Tyk Pro
+
+Select `Keys` from the `System Management` section and click `ADD KEY` as shown in the image below.
+
+![dashboard-keys-page](./img/getting-started-keys.png)
+
+
+You have the option to add your new key to either an existing Policy or an existing individual API. For this tutorial we are going to use an API.
```
Comment (komalsukhani) [inline docs/getting-started.md:346]:
```suggestion
You have the option to create a new key either by selecting an existing Policy created for your API or by simply selecting your API. For this tutorial we are going to use an API.
```

File: docs/getting-started.md [hunk 4]
```diff
@@ -172,3 +173,215 @@ $ curl -i localhost:8080/httpbin/get
...
+
+To select `httpbin` API, you can either:
```
Comment (komalsukhani) [inline docs/getting-started.md:348]:
```suggestion
To select `httpbin` API:
```

Comment (buraksekili) [reply docs/getting-started.md]: Updated 👍

**Pull request merged**